app/src/main/java/com/duckduckgo/app/notification/model/AppFeatureNotification.kt

```diff
@@ -65,3 +70,49 @@ class AppFeatureNotificationSpecification(
     override val description: String = context.getString(descriptionRes)
     override val color: Int = CommonR.color.ic_launcher_red_background
 }
+
+// This is not used at the moment.
+// @ContributesMultibinding(AppScope::class)
```

Comment:
Commented this out as it's not used. Added it as I made changes to the notifications and didn't want to lose the behaviour of this one in case it's ever needed.

Reply:
I would remove this class if not used. It's very easy to find it in git if necessary